Source/WebCore/rendering/style/BasicShapes.h

```diff
@@ -408,46 +435,30 @@ class BasicShapeRect final : public BasicShape {
     const Length& bottom() const { return m_edges.bottom(); }
     const Length& left() const { return m_edges.left(); }
 
-    const LengthSize& topLeftRadius() const { return m_topLeftRadius; }
-    const LengthSize& topRightRadius() const { return m_topRightRadius; }
-    const LengthSize& bottomRightRadius() const { return m_bottomRightRadius; }
-    const LengthSize& bottomLeftRadius() const { return m_bottomLeftRadius; }
-
     void setTop(Length&& top) { m_edges.setTop(WTFMove(top)); }
     void setRight(Length&& right) { m_edges.setRight(WTFMove(right)); }
     void setBottom(Length&& bottom) { m_edges.setBottom(WTFMove(bottom)); }
     void setLeft(Length&& left) { m_edges.setLeft(WTFMove(left)); }
 
-    void setTopLeftRadius(LengthSize&& radius) { m_topLeftRadius = WTFMove(radius); }
-    void setTopRightRadius(LengthSize&& radius) { m_topRightRadius = WTFMove(radius); }
-    void setBottomRightRadius(LengthSize&& radius) { m_bottomRightRadius = WTFMove(radius); }
-    void setBottomLeftRadius(LengthSize&& radius) { m_bottomLeftRadius = WTFMove(radius); }
-
 private:
     BasicShapeRect() = default;
     BasicShapeRect(RectEdges<Length>&&, LengthSize&& topLeftRadius, LengthSize&& topRightRadius, LengthSize&& bottomRightRadius, LengthSize&& bottomLeftRadius);
     BasicShapeRect(Length&& top, Length&& right, Length&& bottom, Length&& left, LengthSize&& topLeftRadius, LengthSize&& topRightRadius, LengthSize&& bottomRightRadius, LengthSize&& bottomLeftRadius);
 
-    Type type() const final { return Type::Rect; }
-
-    Path path(const FloatRect&) const final;
+    Type type() const override { return Type::Rect; }
+    FloatRect rectForBoundingBox(const FloatRect&) const override;
 
     bool canBlend(const BasicShape&) const final;
-    Ref<BasicShape> blend(const BasicShape& from, const BlendingContext&) const final;
+    Ref<BasicShape> blend(const BasicShape& from, const BlendingContext&) const override;
 
-    bool operator==(const BasicShape&) const final;
+    bool operator==(const BasicShape&) const override;
 
-    void dump(TextStream&) const final;
+    void dump(TextStream&) const override;
```

Comment:
Aren't this part of the final class? Not sure why we need to change them to override. Same question for the other final classes

Reply:
`override` and `final` are equivalent if the class itself is marked `final`, so I made them consistent.